gulpfile.js

```diff
@@ -164,7 +168,7 @@ if (!argv.production) {
             configFile: config.report.karma.configFile,
             singleRun: true,
         }, function(exitCode) {
-            done(exitCode);
+            done(exitcode ? new Error('Typescript tests failed') : 0);
```

Comment:
Typo  "function(exitCode){ "  should be "function(exitcode){"
Da sonst folgender Fehler kommt:
done(exitcode ? new Error('Typescript tests failed') : 0);
                 ^
ReferenceError: exitcode is not defined